examples/benchmarks/bubblebench/common/src/view/stage.tsx

```diff
@@ -25,6 +25,7 @@ export const StageView: React.FC<IStageProps> = ({ app }: IStageProps) => {
 		groups.push(
 			<g key={client.color} fill={color} stroke={color}>
 				{client.bubbles.map(({ x, y, r }, index) => {
+					// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Either avoid doing things the linter says to avoid, or document why its necessary and ok to violate the linter enforced polices her.